MeetupPlatformApi/DataTransferObjects/MeetupOutputDto.cs

```diff
@@ -1,15 +1,25 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupOutputDto
 {
+    [Required]
```

Comment:
Are You sure, that Swagger does not mark `int` fields as required by default (looks like `[Required]` attribute is not necessary)